### PR TITLE
[4.0] Add missing license header to passwordstrength.es5.js

### PR DESCRIPTION
**Pull request by HLeithner**
Add the missing MIT License to the file.

### CODE REVIEW

**Comment by HLeithner** [comment]
thx @brianteeman for find this copyright problem.

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on e743f4fa3f22383a5bc03f427606056178347469<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26357">issues.joomla.org/tracker/joomla-cms/26357</a>.</sub>

File: build/media_source/system/js/fields/passwordstrength.es5.js [hunk 1]
```diff
@@ -4,6 +4,28 @@
  * PasswordStrength script by Thomas Kjaergaard
  * License: MIT
```
Comment (Quy) [inline build/media_source/system/js/fields/passwordstrength.es5.js:5]: Similar info as line 8. Remove?

Comment (brianteeman) [reply build/media_source/system/js/fields/passwordstrength.es5.js:5]: no its fine as it is

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on e743f4fa3f22383a5bc03f427606056178347469<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26357">issues.joomla.org/tracker/joomla-cms/26357</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26357">issues.joomla.org/tracker/joomla-cms/26357</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**